moveit_experimental/jog_arm/src/jog_arm/low_pass_filter.cpp

```diff
@@ -64,10 +64,9 @@ double LowPassFilter::filter(double new_measurement)
   previous_measurements_[0] = new_measurement;
```

Comment:
You can drop `previous_measurements_[2]` in the buffer and in reset():
https://github.com/ros-planning/moveit/blob/a0c3595f3180bdaf9d3b7101141dc135e15f763f/moveit_experimental/jog_arm/src/jog_arm/low_pass_filter.cpp#L53

and decrease the size in the header file:
https://github.com/ros-planning/moveit/blob/a0c3595f3180bdaf9d3b7101141dc135e15f763f/moveit_experimental/jog_arm/include/jog_arm/low_pass_filter.h#L58-L59